go.mod

```diff
@@ -19,7 +19,7 @@ require (
 	github.com/jxskiss/base62 v1.1.0
 	github.com/livekit/mageutil v0.0.0-20230125210925-54e8a70427c1
 	github.com/livekit/mediatransportutil v0.0.0-20240302142739-1c3dd691a1b8
-	github.com/livekit/protocol v1.12.0
+	github.com/livekit/protocol v1.11.1-0.20240321081251-69d138dd8fe8
```

Comment:
is `v1.12.0` not compatible?

Reply:
i pointed the diff at the pr version of protocol to make ci pass while i fix all the implementations